### DISCUSSION TITLE
Component testing doesn't work - no tests are found

### DISCUSSION THREAD

**Discussion by paddotk** [cypress-io/cypress#17601] [Component Testing]
Hi, we just set up Cypress and got integration testing working in our project. We also want to use Cypress for unit/component testing, but for some reason no tests can be found. The test runner shows the test file, but doesn't seem to recognize the test in it.

**Cypress 8.1.0
VueJS 2.6.11**

Our setup is this:
```
// tests/e2e/plugins/index.js
module.exports = (on, config) => {
	if (config.testingType === 'component') {
		on('dev-server:start', (options) =>
			startDevServer({ options, webpackConfig })
		)
	}
  
  return config;
}
```

```
// cypress.json
{
	"pluginsFile": "tests/e2e/plugins/index.js",
	"baseUrl": "http://localhost:8080",
	"component": {
		"componentFolder": "src",
		"testFiles": "**/*.test.js"
	}
}
```

```
// our test file within /src
import { mount } from '@cypress/vue';
import OurComponent from './OurComponent.vue';

describe('OurComponent', () => {
	it('renders a message', () => {
		cy.visit('/some/path');
		mount(OurComponent); // Doesn't work without this, either
		cy.get('h1').should('have.text', msg);
	});
});
```

Installed packages:

```
"@cypress/webpack-dev-server": "^1.4.0"
"@cypress/vue": "^2.2.3"
"cypress": "^8.1.0"
"eslint-plugin-cypress": "^2.11.3"
```
I'm probably missing some config rule, but I can't find anything in the docs or elsewhere.

**Comment by paddotk** [comment]
In addition, a co-worker of mine has the same setup and for him it does work. We can't figure why it doesn't work on my pc..

**Comment by JessicaSachs** [comment]
`cy.visit()` is E2E-only. Remove that and it should work.
